examples/mnist_with_visdom.py

```diff
@@ -127,10 +146,15 @@ def validation_inference_function(batch):
     trainer.add_event_handler(TrainingEvents.TRAINING_ITERATION_COMPLETED,
                               get_plot_training_loss_handler(vis, plot_every=log_interval))
 
-    trainer.add_event_handler(TrainingEvents.VALIDATION_COMPLETED, get_log_validation_loss_and_accuracy_handler(logger))
-    trainer.add_event_handler(TrainingEvents.VALIDATION_COMPLETED, get_plot_validation_accuracy_handler(vis))
+    trainer.add_event_handler(TrainingEvents.EPOCH_COMPLETED, Validate(val_loader, epoch_interval=1))
+
+    trainer.add_event_handler(TrainingEvents.VALIDATION_COMPLETED,
+                              get_log_validation_loss_and_accuracy_handler(logger, val_loader))
+    trainer.add_event_handler(TrainingEvents.VALIDATION_COMPLETED, get_plot_validation_loss_handler(vis))
+    trainer.add_event_handler(TrainingEvents.VALIDATION_COMPLETED,
+                              get_plot_validation_accuracy_handler(vis, val_loader))
     trainer.add_event_handler(TrainingEvents.VALIDATION_COMPLETED, lambda trainer: trainer.validation_history.clear())
```

Comment:
This line can be removed as the `Validate` handler will now clear this before each run